kpi/models/asset.py

```diff
@@ -390,6 +392,16 @@ def _rename_translation(self, content, _from, _to):
             raise ValueError('Duplicate translation: {}'.format(_to))
         _ts[_ts.index(_from)] = _to
 
+    def _contains_invalid_chars(self, content):
+        for row in content['survey']:
+            try:
+                if row['default'] and bool(re.search(
+                        r'[<|>|&]', row['default'])):
+                    raise ValidationError(
+                        'XForm questions settings may contain malicious content')
```

Comment:
Do we usually get such strings from a constants/strings module/file?

Reply:
I dont believe so. The error message string is passed to the Exception.

Reply:
What I was asking was whether this project has something similar to `common_tags` (onadata.libs.utils.common_tags) in Onadata and whether the convention is to use it or not.

Reply:
No, unfortunately I have not found anything similar to Onadata's `common_tag.py` file.
The repository does have a `constants.py` file, but from the looks of it, it wouldn't be a correct fit to place this string value.